slateruhf.py

```diff
@@ -17,9 +17,15 @@ class PySCFSlaterUHF:
     def __init__(self,nconfig,mol,mf):
         self.occ=np.asarray(mf.mo_occ > 0.9)
         self.parameters={}
-        
-        self.parameters['mo_coeff_alpha']=mf.mo_coeff[0][:,self.occ[0]]
-        self.parameters['mo_coeff_beta']=mf.mo_coeff[1][:,self.occ[1]]
+
+        #Determine if we're initializing from an RHF or UHF object...
+        if len(mf.mo_occ.shape)==2:        
+          self.parameters['mo_coeff_alpha']=mf.mo_coeff[0][:,self.occ[0]]
+          self.parameters['mo_coeff_beta'] =mf.mo_coeff[1][:,self.occ[1]]
+        else:
+          self.parameters['mo_coeff_alpha']=mf.mo_coeff[:,self.occ]

```

Comment:
Can you make this work also for ROHF? the alpha gets mf.mo_occ > 1.1 and beta gets mf.mo_occ > 0.9